### PR TITLE
Cleanup and add new Spanish translations

### PR DESCRIPTION
**Pull request by NotFenixio**

### CODE REVIEW

**Comment by NotFenixio** [comment]
Ready to merge!

File: src/i18n/es.ts [hunk 1]
```diff
@@ -171,49 +159,93 @@ export const es = {
...
+  noButton: "No",
+  dialogTitle_alert: "Alerta",
+  dialogTitle_prompt: "Solicitud",
+  dialogTitle_confirm: "Confirmar",
+  reportReasonPlaceholder: "Motivo del reporte",
+  newPasswordPlaceholder: "Nueva contraseña",
+  oldPasswordPlaceholder: "Contraseña actual",
+  deleteAccountPasswordPlaceholder: "Contraseña",
+  routeRelated: "Más",
+  livechat: "Chat en vivo",
```
Comment (mybearworld) [inline src/i18n/es.ts:221]: Just to make sure, does that convey "live chat" and not "life chat"?

Comment (NotFenixio) [reply src/i18n/es.ts:221]: Yep, "en vivo" is live, if you were to convey "life chat" it'd be "Chat vivo".

Comment (mybearworld) [reply src/i18n/es.ts:221]: Alright, great!

**Pull request merged**